app/controllers/books_controller.rb

```diff
@@ -10,4 +10,34 @@ def show
       redirect_to books_path
     end
   end
-end
\ No newline at end of file
+
+  def new
+    @book = Book.new
+  end
+
+  def create
+    book = Book.new(
+      year_published: book_params[:year_published],
+      page_count: book_params[:page_count],
+      thumbnail: book_params[:thumbnail].presence || 'https://nnp.wustl.edu/img/bookCovers/genericBookCover.jpg',
+      title: book_params[:title].titlecase)
+    if book.save
+      author_input = book_params[:authors].split(",")
```

Comment:
```suggestion
      author_name_input = book_params[:authors].split(",")
```
(similar for usage below)

Reply:
Agreed, fixed on branch.